### PR TITLE
Added rubygem-hammer_cli_foreman_docker (0.0.3)

### PR DESCRIPTION
**Pull request by mbacovsky**
Scratch build: http://koji.katello.org/koji/taskinfo?taskID=234167

### CODE REVIEW

**Comment by elyezer** [comment]
I have installed from koji and this package is installing good. I did not have the hammer docker command before installing it and it was available after the installation. Have run some docker subcommands and they are working properly, but I was not able to validate the container subcommand because they still not ready yet.

File: comps/comps-foreman-plugins-fedora19.xml [hunk 1]
```diff
@@ -41,6 +41,7 @@
       <packagereq type="default">rubygem-hammer_cli_foreman_bootdisk</packagereq>
       <packagereq type="default">rubygem-hammer_cli_foreman_discovery</packagereq>
       <packagereq type="default">rubygem-hammer_cli_foreman_salt</packagereq>
+      <packagereq type="default">rubygem-hammer_cli_foreman_docker</packagereq>
```
Comment (domcleal) [inline comps/comps-foreman-plugins-fedora19.xml]: Nitpick: not alphabetical (ditto other files)

File: rubygem-hammer_cli_foreman_docker/rubygem-hammer_cli_foreman_docker.spec [hunk 2]
```diff
@@ -0,0 +1,70 @@
+%global gem_name hammer_cli_foreman_docker
+%global confdir hammer
+
+Summary: Foreman Docker-related commands for Hammer
+Name: rubygem-%{gem_name}
+Version: 0.0.3
+Release: 1%{?dist}
+Group: Applications/System
+License: GPLv3
```
Comment (domcleal) [inline rubygem-hammer_cli_foreman_docker/rubygem-hammer_cli_foreman_docker.spec]: GPLv3+

File: rubygem-hammer_cli_foreman_docker/rubygem-hammer_cli_foreman_docker.spec [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+Requires: ruby(rubygems)
+Requires: rubygem(hammer_cli_foreman) >= 0.1.2
+BuildRequires: ruby(rubygems)
+BuildRequires: rubygems-devel
+BuildRequires: ruby
+BuildArch: noarch
+Provides: rubygem(%{gem_name}) = %{version}
+
+%description
+Foreman docker plugin for Hammer CLI provides docker-related commands on command line
```
Comment (domcleal) [inline rubygem-hammer_cli_foreman_docker/rubygem-hammer_cli_foreman_docker.spec]: line wrap at 80 chars

**Comment by domcleal** [comment]
For the future (no need to release a new gem for it), can you also fix a couple of licence related items in the gemspec?  http://projects.theforeman.org/issues/9748

**Comment by mbacovsky** [comment]
@domcleal, thanks for review. I've fixed the issues from comments. In next gem release I'll include updated locales and license. Isn't missing license a blocker for RPM package?

**Comment by domcleal** [comment]
Yeah it's not great, but I trust that you'll do it for the next release and it is in the package metadata for the time being.  Once added to the gemspec it'll be in the next RPM.

**Comment by domcleal** [comment]
Thanks @mbacovsky, will publish to nightly and 1.8 repos.

**Pull request merged**